network/network_windows.go

```diff
@@ -297,11 +298,18 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *EndpointInfo, extIf *exter
 		return nil, errNetworkModeInvalid
 	}
 
+	// DelegatedNIC flag: hcn.DisableHostPort(1024)
 	if nwInfo.NICType == cns.DelegatedVMNIC {
 		hcnNetwork.Type = hcn.Transparent
 		hcnNetwork.Flags = hcn.DisableHostPort
 	}
 
+	// AccelnetNIC flag: hcn.EnableIov(9216)
+	if nwInfo.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
+		hcnNetwork.Type = hcn.Transparent
+		hcnNetwork.Flags = hcnIov // TODO: pick up latest hcsshim release for hcnIov flag when it's ready
```

Comment:
no need to set `hcnNetwork.Flags = hcn.DisableHostPort`  for acclnet?

Reply:
no need; we need to do hcnNetwork.Flags = hcn.DisableHostPort | hcn.EnableIov but hcsshim is not ready yet, so now we just do this as a workaround

Reply:
can you add workitem for it so that we would not lose track on this

Reply:
sure, I am currently tracking this issue now